packages/sdk/src/libs/coingecko/coingecko.ts

```diff
@@ -0,0 +1,47 @@
+import axios from "axios";
+import assert from "assert";
+import { get } from "lodash";
+
+class Coingecko {
+  private host: string;
+  constructor(host = "https://api.coingecko.com") {
+    this.host = host;
+  }
+  // Fetch historic prices for a `contract` denominated in `currency` between timestamp `from` and `to`. Note timestamps
+  // are assumed to be js timestamps and are converted to unixtimestamps by dividing by 1000.
+  async getHistoricContractPrices(contract: string, from: number, to: number, currency = "usd") {
+    assert(contract, "requires contract address");
+    assert(currency, "requires currency symbol");
+    assert(from, "requires from timestamp");
+    assert(to, "requires to timestamp");
+    from = from / 1000;
```

Comment:
Do you need to `Math.round/floor/ceil` these at all?

Reply:
good call